### DISCUSSION TITLE
@vkontakte/vkui-lab

### DISCUSSION THREAD

**Discussion by ukqfhf** [VKCOM/VKUI#5976] [Ideas] [answered]
Что если вынести все экспериментальные компоненты из основной библиотеки в lab? В документации также разделить стабильные компоненты от экспериментальных как это сделано в MUI.

Референцы:

* [lab из MUI](https://mui.com/material-ui/about-the-lab/)

**Comment by BlackySoul** [comment, marked as answer]
Привет, мы [приняли решение](https://github.com/VKCOM/VKUI/issues/3581), что вместо отдельного пакета или импорта будем поставлять все через единую точку входа, но с префиксом `unstable_*`. Кажется, что так удобнее и не путает никого, в будущем будет проще перейти на стабильную версию, просто отбросив префикс. У наших нестабильных компонентов тоже есть [отдельная](https://vkcom.github.io/VKUI/#/Unstable) страничка, и в документации они помечены специальным значком:

<img width="327" alt="image" src="https://github.com/VKCOM/VKUI/assets/7431217/80e56508-de1f-43b1-a84f-88427fefd963">
<img width="1130" alt="image" src="https://github.com/VKCOM/VKUI/assets/7431217/890eeb70-def3-4e89-b4bf-337e0ae54f0b">
